### ISSUE TITLE
Lowercase ASCII k and s

### ISSUE THREAD

**Issue by dougmoscrop** [mathiasbynens/swapcase#1]
Is it intended that the lowercase letters `s` and `k` convert to something other than `S` and `K`?

See swap-map.json

```
"k": "\u212A",
"s": "\u017F",
```

Probably not going to break anything for anyone, but it's still a bit strange.

**Comment by stuartpb** [comment]
Yeah, this module seems to hinge on a misconception of what case folding does; it's a one-way transformation, where multiple inbound nodes can point to the same outbound node.

It should be using the upper-case transformations for lower-case characters combined with the lower-case transformations for upper-case characters: this is not the same thing as the folded-case transformations.

It's also wrong about using full-folding transformations (which are similar to a compatibility normalization) over simple ones (which are similar to a canonical composition normalization), but noting that is kind of irrelevant when the core concept of this module is so entirely busted.

See http://unicode.org/faq/casemap_charprop.html

**Comment by stuartpb** [comment]
Why was this issue closed? It's still a problem.

**Comment by dougmoscrop** [comment]
Because I did some clean up all old/open issues that maintainers show no sign of addressing.
